apps/project/admin.py

```diff
@@ -25,23 +25,22 @@
 TRIGGER_LIMIT = 5
 
 
-def trigger_project_stat_calc(modeladmin, request, queryset):
-    generator = generate_entry_stats if modeladmin == ProjectEntryStatsAdmin else generate_ary_stats
-    for project_id in queryset.values_list('project_id', flat=True).distinct()[:TRIGGER_LIMIT]:
-        generator.delay(project_id, force=True)
-    messages.add_message(
-        request, messages.INFO,
-        mark_safe(
-            f'Successfully triggered Project Stats Calculation ({generator.__name__}) for projects: <br><hr>' +
-            '<br>'.join(
-                '* {0} : {1}'.format(*value)
-                for value in queryset.values_list('project_id', 'project__title').distinct()[:TRIGGER_LIMIT]
+def trigger_project_stat_calc(generator):
```

Comment:
Was this a separate pull request in develop?

Reply:
I think so

Reply:
Yes, the other pull request is exactly that